ja/lessons/ecto/changesets.md

```diff
@@ -27,61 +27,62 @@ iex> %Ecto.Changeset{}
 チェンジセットを本当に役立つものにするためには、作成時に、どのようなデータになるかという設計図を提供する必要があります。
 フィールドと型の定義を持つ私たちが作ったスキーマよりも、データのためのより良い設計図とはなんでしょうか？
 
-一般的な `User` スキーマをみてみましょう:
+前のレッスンの `Friends.Person` スキーマを使いましょう:
 
 ```elixir
-defmodule User do
+defmodule Friends.Person do
   use Ecto.Schema
 
-  schema "users" do
-    field(:name, :string)
+  schema "people" do
+    field :name, :string
+    field :age, :integer, default: 0
   end
 end
 ```
 
-`User` スキーマを使うチェンジセットを作るためには、 `Ecto.Changeset.cast/4` を使います:
+`Person` スキーマを使うチェンジセットを作るためには、 `Ecto.Changeset.cast/4` を使います:
 
 ```elixir
-iex> Ecto.Changeset.cast(%User{name: "Bob"}, %{}, [:name])
-#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #User<>,
+iex> Ecto.Changeset.cast(%Friends.Person{name: "Bob"}, %{}, [:name, :age])
+#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #Friends.Person<>,
  valid?: true>
 ```
 
-最初のパラメータは元のデータで、この場合は空の `%User{}` 構造体です。
+最初のパラメータは元のデータで、この場合は初期化された `%Friends.Person{}` 構造体です。
 Ectoは構造体そのものに基づいてスキーマを見つけることができます。
 2番目は私たちが行いたい変更であり、ただの空のマップです。
 3番目のパラメータが `cast/4` を特別なものにします。これは通過させることを許可するフィールドのリストであり、これによってどのフィールドが変更可能なのかを制御可能とし、残りを安全に保護します。
 
 ```elixir
-iex> Ecto.Changeset.cast(%User{name: "Bob"}, %{"name" => "Jack"}, [:name])
+iex> Ecto.Changeset.cast(%Friends.Person{name: "Bob"}, %{"name" => "Jack"}, [:name, :age])
 #Ecto.Changeset<
- action: nil,
- changes: %{name: "Jack"},
- errors: [],
- data: #User<>,
- valid?: true
+  action: nil,
+  changes: %{name: "Jack"},
+  errors: [],
+  data: #Friends.Person<>,
+  valid?: true
 >
 
-iex> Ecto.Changeset.cast(%User{name: "Bob"}, %{"name" => "Jack"}, [])
-#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #User<>,
-valid?: true>
+iex> Ecto.Changeset.cast(%Friends.Person{name: "Bob"}, %{"name" => "Jack"}, [])
+#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #Friends.Person<>,
+ valid?: true>
 ```
 
 2回目では新しいnameが明示的に許可されていないため、無視されていることがわかるでしょう。
 
 `cast/4` の代わりとして `change/2` もあり、これは `cast/4` のように変更をフィルタリングする機能を持ちません。
 これは変更を加えるソースが信頼できるとき、あるいは手動でデータを扱う時に便利です。
 
-ここではチェンジセットを作りましたが、バリデーションを持っていないので、ユーザーのあらゆる名前の変更が受け付けられてしまい、その結果空の名前を持つ可能性もあります。
+ここではチェンジセットを作りましたが、バリデーションを持っていないので、人のあらゆる名前の変更が受け付けられてしまい、その結果空の名前を持つ可能性もあります。
```

Comment:
maybe)
```suggestion
ここではチェンジセットを作りましたが、バリデーションを持っていないので、友人のあらゆる名前の変更が受け付けられてしまい、その結果空の名前を持つ可能性もあります。
```

🤔

Reply:
> Now we can create changesets, but since we do not have validation, any changes to person's name will be accepted, so we can end up with an empty name:

If we translate this word, it doesn't seem like a person object. It looks like a sentence in another context. I'll just leave it if It was my work.

```suggestion
ここではチェンジセットを作りましたが、バリデーションを持っていないので、personのnameにあらゆる変更が受け付けられしまい、その結果空の名前をなる可能性もあります。
```

Reply:
@marocchino https://github.com/elixirschool/elixirschool/pull/2028/commits/bcf38e2938d28308d6cd2b58602ff059c1d148e7

💪